examples/resources/biganimal_cluster/ha/resource.tf

```diff
@@ -68,6 +68,12 @@ resource "biganimal_cluster" "ha_cluster" {
     size              = "4 Gi"
   }
 
+  #  wal_storage = {
+  #    volume_type       = "gp3"
+  #    volume_properties = "gp3"
```

Comment:
should we handle setting volume properties internally since it does not have properties like az
` #    volume_type       = "azurepremiumstorage"
  #    volume_properties = "P1"`

Reply:
oh for AWS/GCP? Possibly it could work but maybe we can do it in another PR as an enhancement so that they can remove volume_properties for AWS/GCP. As of right now I think I recall in the API(for BA anyways) the volume_properties have to match the volume_type otherwise it will throw an error